recipes/robin-hood-hashing/all/conanfile.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+import os
+
+from conans import ConanFile, tools
+
+class ConanRecipe(ConanFile):
+    name = "robin-hood-hashing"
+    description = "Faster and more efficient replacement for std::unordered_map / std::unordered_set"
+    topics = ("conan", "robin-hood-hashing", "header-only", "containers")
+    homepage = "https://github.com/martinus/robin-hood-hashing"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "MIT"
+
```

Comment:
```suggestion
    license = "MIT"
    settings = "compiler"
```

Reply:
Why do we need compiler? Conan docs says we can skip `settings` for header-only..?

Reply:
Oh it's for C++ standard check...